### PR TITLE
[OSCD] - Runscripthelper.exe runs script (LoLBin)

### PR DESCRIPTION
**Pull request by stvetro**
Task 81 from #1014

### CODE REVIEW

File: rules/windows/process_creation/win_susp_runscripthelper.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSBinaries/Runscripthelper.yml
+author: Victor Sergeev, oscd.community
+date: 2020/10/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    image_path:
+        Image|endswith: 'Runscripthelper.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_susp_runscripthelper.yml:14]: could you please add `\` here as well?

Comment (stvetro) [reply rules/windows/process_creation/win_susp_runscripthelper.yml]: done

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_susp_runscripthelper.yml]: Thanks!

**Comment by yugoslavskiy** [comment]
Well done, @stvetro! Thank you very much!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**